### PR TITLE
elm-format 0.8.7

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by avh4** [comment]
Ah, apparently brew was building the unoptimized build of elm-format, and that's failing now because the build scripts changed.

To correctly build the release build, the command should be:

```
dev/build.sh _build/bin/elm-format/O2/elm-format
```

And the resulting binary will be at `_build/bin/elm-format/O2/elm-format` instead of at `_build/elm-format`

**Comment by avh4** [comment]
Also re the comment ``Directly running `cabal v2-install` fails: Invalid file name in tar archive: "avh4-lib-0.0.0.1/../"`` I believe that's now fixed upstream.

**Comment by p-linnane** [comment]
Thank you for hopping in here @avh4. I was unable to get this build to suceed with `cabal v2-install`, so I just left that comment and updated the build script section per your suggestion.

File: Formula/elm-format.rb [hunk 1]
```diff
@@ -42,8 +43,8 @@ def install
 
     # Directly running `cabal v2-install` fails: Invalid file name in tar archive: "avh4-lib-0.0.0.1/../"
     # Instead, we can use the upstream's build.sh script, which utilizes the Shake build system.
```
Comment (chenrui333) [inline Formula/elm-format.rb:45]: maybe remove this line?

Comment (p-linnane) [reply Formula/elm-format.rb:45]: The build still fails when trying ot use `cabal v2-install`. Unless I'm misunderstanding which line you're referring to.

**Review by carlocab — APPROVED** [review]

File: Formula/elm-format.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/elm-format.rb:45]: Oh, i see, i thought we are using `hpack` for the build now. never mind then :)

**Review by chenrui333 — APPROVED** [review]

File: Formula/elm-format.rb (hunk 1 above)
Comment (p-linnane) [reply Formula/elm-format.rb:45]: The build script uses `hpack` now 👍

**Pull request merged**